### ISSUE TITLE
Where to pass bearer token?

### ISSUE THREAD

**Issue by ion-elgreco** [samster25/unitycatalog-python#4]
Wanted to see if I could replace my internally built UC API wrapper but using this package I cannot figure out how to pass a bearer token?

Tried passing the below to extra_headers, but no luck unfortunately.

```
headers = {
  "Authorization": f"Bearer {my_token}",
  "Content-Type": "application/json",
}
```

**Comment by samster25** [comment]
[This](https://github.com/Eventual-Inc/Daft/pull/2377/files#diff-7ce2b8235cdfe5e85b903c8ef2c7218261371db50fbe1ce5ed852a9ef13038adR32) is how we did it in daft using this package.
```
unitycatalog.Unitycatalog(
            base_url=endpoint.rstrip("/") + "/api/2.1/unity-catalog/",
            default_headers={"Authorization": f"Bearer {token}"},
        )
```

What sort of error are you getting?

**Comment by ion-elgreco** [comment]
Ah didn't realize that you have to explicitly add /api/2.1/unity-catalog, in my own wrapper this was automatically appended to the uri

**Comment by samster25** [comment]
@ion-elgreco Yeah currently this project is just generated from the openapi stubs. I think it would be a good idea to build a higher level api that we can also expose.

We could move over the Daft code to here: 
https://github.com/Eventual-Inc/Daft/blob/d2107b75966eed116403123e97ef29abdd0db8ec/daft/unity_catalog/unity_catalog.py#L12
